static/src/javascripts/bootstraps/enhanced/football.js

```diff
@@ -346,14 +346,17 @@ const init = () => {
         }
     );
 
-    bean.on(
-        document.body,
-        'change',
-        $('form.football-leagues')[0],
-        function onChange() {
-            window.location = this.value;
-        }
-    );
+
+    if ( $('form.football-leagues')[0]) {
```

Comment:
I'd imagine you've thought of this but just to double check - could this ever cause an issue where we're looking for an index `[0]` of `undefined`?

Reply:
I also think it'd be safer if we check it has value `if ($('form.football-leagues') && $('form.football-leagues')[0])`

Reply:
Agreed, I'll amend